CMakeLists.txt

```diff
@@ -52,17 +52,16 @@ add_custom_target(Download)
 #
 # Do we want to build in Python support where available
 #
-option(FLETCH_BUILD_WITH_PYTHON "Build with Python support where appropriate" FALSE)
-mark_as_advanced(FLETCH_BUILD_WITH_PYTHON)
+option(fletch_BUILD_WITH_PYTHON "Build with Python support where appropriate" FALSE)
 
 #
 # Convenience Option for Dashboards to turn on ALL available Packages
 #
 option(fletch_ENABLE_ALL_PACKAGES "Enable all available packages" FALSE)
 
-if (FLETCH_BUILD_WITH_PYTHON)
-    find_package(PythonInterp)
-    find_package(PythonLibs)
+if (fletch_BUILD_WITH_PYTHON)
+  find_package(PythonInterp)
```

Comment:
I think you should add the version requirement here and remove the duplicated find_package found in External_Caffe.cmake, CMake/External_VTK.cmake and anywhere else the logic is duplicated.